packages/components/components/elvis-progress-linear/src/react/elvia-progress-linear.tsx

```diff
@@ -39,8 +30,8 @@ export const ProgressLinear: FC<ProgressLinearProps> = ({
       aria-valuemax={100}
       role={ariaRole}
       id={componentId}
-      aria-label={ariaLabel ?? computedAriaLabel}
-      aria-valuetext={ariaValueText ?? computedAriaValueText}
+      aria-label={(ariaLabel ?? lang === 'no') ? 'Progresjon' : 'Progress'}
```

Comment:
I don't think this will do what you want! Check the placement of the parentheses 👀 
`ariaLabel ?? (lang === 'no' ? 'Progresjon' : 'Progress')`

Reply:
I found out that if you click ctrl + S after writing the sentence like this:
`{(ariaValueText ?? lang === 'no' ? 'Progresjonen er nå på ' : 'The progress is now at ')}` 

It would default to this:
`{(ariaLabel ?? lang === 'no') ? 'Progresjon' : 'Progress'}` Which is wrong as you said.

I have corrected the sentence to the following code, which now isolates the handling of default `no` and `en` values separately:
`{ariaValueText ?? (lang === 'no' ? 'Progresjonen er nå på ' : 'The progress is now at ')}`